### PR TITLE
CLOUD-61480 - use greatest and latest from image catalog for dev versions

### PR DESCRIPTION
**Pull request by sodre90**
@keyki pls review

### CODE REVIEW

File: core/src/main/java/com/sequenceiq/cloudbreak/service/image/HdpInfoSearchService.java [hunk 1]
```diff
@@ -6,11 +6,11 @@
 
 import javax.inject.Inject;
 
+import org.apache.commons.lang3.StringUtils;
```
Comment (akanto) [inline core/src/main/java/com/sequenceiq/cloudbreak/service/image/HdpInfoSearchService.java]: why did you switch from org.springframework.util.StringUtils to apache Stringutils?

**Comment by keyki** [comment]
Is it supposed to go to master rather than rc-1.5?

**Comment by sodre90** [comment]
@keyki the master is the dev version, 1.5 is rc branch, so it is OK on master

**Comment by sodre90** [comment]
@akanto I rewrited to spring version of StringUtils

**Pull request merged**